### PR TITLE
[Twint] Alert presentation fix

### PR DESCRIPTION
**Pull request by goergisn**
## Summary
- Made `PresentableComponentWrapper.requiresModalPresentation` variable so a component can present UIViewControllers without wrapping them into a sheet (needed for UIAlertController)
- Cancelling when Twint does not find installed apps

Simulator Screenshot - iPhone 14 - 2024-03-01 at 14 57 01{width=300}

### CODE REVIEW

File: AdyenActions/Components/SDK/TwintSDKActionComponent.swift [hunk 1]
```diff
@@ -121,10 +122,15 @@ import Foundation
...
-            alert.addAction(.init(
-                title: localizedString(.dismissButton, configuration.localizationParameters),
-                style: .default
-            )
+            alert.addAction(
+                .init(
+                    title: localizedString(.dismissButton, configuration.localizationParameters),
+                    style: .default, 
+                    handler: { _ in
+                        // TODO: Is this the right thing to do here? - we have to let drop-in know to hide the spinner without dismissing drop-in
```
Comment (goergisn) [inline AdyenActions/Components/SDK/TwintSDKActionComponent.swift:130]: Any opinion on this part here?
I believe this is the right thing to do to stop drop-in from loading

Comment (neelSharma12) [reply AdyenActions/Components/SDK/TwintSDKActionComponent.swift:130]: you mean to hide the spinner next to  twint PM?

Comment (goergisn) [reply AdyenActions/Components/SDK/TwintSDKActionComponent.swift:130]: Yes exactly. The dismissing of the alert should just hide the spinner but not dismiss drop-in I assume.

Comment (neelSharma12) [reply AdyenActions/Components/SDK/TwintSDKActionComponent.swift:130]: yes we should do this :)

Comment (goergisn) [reply AdyenActions/Components/SDK/TwintSDKActionComponent.swift:130]: Ok I will get rid of the discussion todo 👌

**Pull request merged**